### PR TITLE
newdevice: add support for Moxa EDS-P510A-8PoE 

### PR DESCRIPTION
**Pull request by priiduonu**
Support PoE port consumption, SFP module monitoring and power input statuses for Moxa EDS-P510A-8PoE industrial switch.

Add Moxa EDS-P510A-8PoE MIB.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: mibs/moxa/MOXA-EDSP510A_8POE-V3.7-MIB.my [hunk 1]
```diff
@@ -0,0 +1,4880 @@
+
```
Comment (laf) [inline mibs/moxa/MOXA-EDSP510A_8POE-V3.7-MIB.my]: Can you rename this file to MOXA-EDSP510A8POE-MIB

File: includes/definitions/discovery/moxa-etherdevice.yaml [hunk 2]
```diff
@@ -14,3 +18,61 @@ modules:
                     descr: 'Power Consumption'
                     index: 'powerConsumption.{{ $index }}'
                     value: powerConsumption
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::poePortConsumption
```
Comment (laf) [inline includes/definitions/discovery/moxa-etherdevice.yaml]: This should just be `oid: poePortConsumption`

Add MOXA-EDSP510A8POE-MIB at the top like:

```
mib: MOXA-EDSP510A8POE-MIB
```

Comment (laf) [reply includes/definitions/discovery/moxa-etherdevice.yaml]: goes for all the others in this file.

File: includes/definitions/discovery/moxa-etherdevice.yaml [hunk 3]
```diff
@@ -14,3 +18,61 @@ modules:
...
+                    descr: '{{ $ifDescr }} SFP module Receive Power'
+                    index: 'sfpRxPower.{{ $index }}'
+                    value: sfpRXPower
+        state:
+            data:
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::power1InputStatus
+                    num_oid: .1.3.6.1.4.1.8691.7.86.1.10.1.
+                    descr: 'Power Input 1 Status'
+                    index: 'powerInputStatus.1'
```
Comment (laf) [inline includes/definitions/discovery/moxa-etherdevice.yaml]: May as well change this to `index: 'powerInputStatus.{{ $index }}`'

File: includes/definitions/discovery/moxa-etherdevice.yaml [hunk 4]
```diff
@@ -14,3 +18,61 @@ modules:
...
+                    value: power1InputStatus
+                    state_name: powerInputStatus
+                    states:
+                        - { value:  0, generic: 2, graph: 0, descr: not present }
+                        - { value:  1, generic: 0, graph: 0, descr: present }
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::power2InputStatus
+                    num_oid: .1.3.6.1.4.1.8691.7.86.1.10.2.
+                    descr: 'Power Input 2 Status'
+                    index: 'powerInputStatus.2'
```
Comment (laf) [inline includes/definitions/discovery/moxa-etherdevice.yaml]: Same here

File: includes/definitions/discovery/moxa-etherdevice.yaml (hunk 3 above)
Comment (priiduonu) [reply includes/definitions/discovery/moxa-etherdevice.yaml]: Input 1 and Input 2 have different OID-s and both have index 0 so it probably won't work.

**Comment by laf** [comment]
Thanks for updating. Just test data now :) https://docs.librenms.org/#Developing/os/Test-Units/#example-workflow

**Comment by priiduonu** [comment]
Added test data.

**Comment by priiduonu** [comment]
Renamed the MIB file.

**Pull request merged**